app/src/main/java/com/android/PetPamper/StringSanitizer.kt

```diff
@@ -0,0 +1,109 @@
+package com.android.PetPamper
+
+import java.util.regex.Pattern
```

Comment:
Combine multiple replace calls into a single regex replacement where possible.
here's a suggestion of how we can do it 
```suggestion

object StringSanitizer {
  private val emailPattern = Regex("""[^a-zA-Z0-9.!#$%&'*+/=?^_`{|}~@-]""")
  private val redundantDotPattern = Regex("""\.{2,}""")
  private val phonePattern = Regex("""[^0-9]""")
  private val floatPattern = Regex("""[^0-9,.]""")
  private val intPattern = Regex("""[^0-9]""")

  fun sanitizeEmail(str: String): String {
    return str.replace(emailPattern, "").replace(redundantDotPattern, ".")
  }

  fun sanitizePhone(str: String): String {
    return if (str.startsWith("+")) str.replaceFirst("+", "00").replace(phonePattern, "") else str.replace(phonePattern, "")
  }

  fun sanitizeFloat(str: String): String {
    return str.replace(Regex("""[,'.]"""), "").replace(',', '.')
  }

  fun sanitizeInt(str: String): String {
    return str.replace(intPattern, "")
  }
}

```